config/http_config.go

```diff
@@ -154,10 +154,12 @@ func NewRoundTripperFromConfig(cfg HTTPClientConfig, name string, disableKeepAli
 				conntrack.DialWithName(name),
 			),
 		}
-		// TODO: use ForceAttemptHTTP2 when we move to Go 1.13+.
-		err := http2.ConfigureTransport(rt.(*http.Transport))
-		if err != nil {
-			return nil, err
+		if enableHTTP2 {
```

Comment:
Can we add a comment that this should go away once Go is fixed?

Reply:
Actually  if we add a comment, could we tight the disabling to the disableKeepAlives arg?

Reply:
In theory yes, but it feels pretty hacky.